libs/external-router/driver/src/public_api.ts

```diff
@@ -6,3 +6,4 @@ export {
   DaffExternalRouterDriverInterface,
   DaffExternalRouterDriver,
 } from './interfaces/external-router-driver.interface';
+export { NOT_FOUND_RESOLUTION } from './not-found-resolution';
```

Comment:
this should have a daff or magento prefix since its exported

Reply:
28c849b8